spec/classes/init_spec.rb

```diff
@@ -48,7 +48,7 @@
           end
 
           case [facts[:os]['family'], facts[:os]['release']['major']]
-          when %w[RedHat 7], %w[RedHat 9], %w[Debian 12]
+          when %w[RedHat 7], %w[RedHat 9], %w[Debian 12], %w[Fedora \d+]
```

Comment:
I think the `\d+` doesn't match. Did you try
```suggestion
          when %w[RedHat 7], %w[RedHat 9], %w[Debian 12], %w[Fedora /\d+/]
```

But even then, I think `%w[a b]` always produces strings, so you'd need to be explicit and say
```suggestion
          when %w[RedHat 7], %w[RedHat 9], %w[Debian 12], %w[Fedora 37],  %w[Fedora 38]
```

Reply:
Wait, lol, no!

`family` is `RedHat` for Fedora too. So how about

```suggestion
          when %w[RedHat 7], %w[RedHat 9..999], %w[Debian 12]
```

Reply:
Adding my brain dump from IRC:
```
19:19 < Zhenech> ooooh
19:20 < Zhenech> that major not mathing 9..999 is because it's a string, not an int
19:21 < Zhenech> mh, no, even that doesn't help
19:21 < Zhenech> fuckit
19:24 < Zhenech> okay, yeah, it constructs an Array, and that Array, as an object, must match, it can't contain matchers itself
```